memory/training/utils/dataset.py

```diff
@@ -9,6 +9,7 @@
 import random
 
 import skimage
+import skimage.io
```

Comment:
This doesn't seem to be used?

Reply:
At least on my machine, when `skimage` is imported it doesn't automatically import the `io` module---that has to be imported separately. I can't remember where exactly but I think `skimage.io` definitely is used somewhere in the code because python yelled at me when I tried to run the script without importing the extra module.